ffw/BasicCommunicationRPC.js

```diff
@@ -901,6 +954,18 @@ FFW.BasicCommunication = FFW.RPCObserver
         this.client.send(JSONMessage);
       },
       /**
+       * notify SDL about readiness to provide system time.
+       * BasicCommunication should be sunscribed to this notification
```

Comment:
@pvvasilev, thanks, fixed in [43c9ff7](https://github.com/CustomSDL/sdl_hmi/pull/4/commits/43c9ff7ae8ada9211f2ed5ef1b79da294f8478ed)